_layouts/frontpage.html

```diff
@@ -16,7 +16,6 @@
         </p>
         <span class="hide"><i class="fa fa-close"></i></span>
       </div>
-      {% include scala-days-sponsors.html %}
```

Comment:
Leave a comment that this is where that include should be reintroduced in the future?